content/src/Environment.ts

```diff
@@ -357,6 +360,16 @@ export class EnvironmentBuilder {
       () => process.env.REPOSITORY_QUEUE_MAX_QUEUED ?? 50
     )
 
+    /*
+     * These are configured as 'CACHE_{CACHE_NAME}_{ENTITY_TYPE}=MAX_SIZE'.
```

Comment:
Should this key be something more specific like `ENTITIES_CACHE`? Just in case we have another cache configuration in the future.

Reply:
In this line I'm explaining the schema. The `{CACHE_NAME}` for the cache that is added in this PR is `ENTITIES_BY_POINTERS`. It's defined in `CacheManager`

Reply:
Okay, my doubt was because it is filtering all environment variables that start with only `CACHE`